tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -1602,7 +1602,7 @@ public function test_remove_insecure_properties_removes_unsafe_styles() {
 						),
 						'core/cover'  => array(
 							'filter' => array(
-								'duotone' => 'var(--wp--preset--duotone--blue-red, var(--fallback-unsafe))',
```

Comment:
These changes to the `WP_Theme_JSON` tests are similar to https://github.com/WordPress/gutenberg/pull/31982.

We were relying on the fact `safecss_filter_attr` doesn't support nested `var()` functions in order to test that attributes were being passed through `safecss_filter_attr`. Since this PR makes it so that `safecss_filter_attr` supports nested `var()` functions, we need to change the test cases to a different invalid CSS string.

cc. @oandregal to make sure I'm not mistaken 😀

Reply:
Yeah, changes make sense. Thanks!